pkg/client_test.go

```diff
@@ -23,7 +24,8 @@ func TestWithConfig(t *testing.T) {
 
 	client, err := prefab.NewClient(
 		prefab.WithConfigs(configs),
-		prefab.WithInitializationTimeoutSeconds(0.5))
+		prefab.WithInitializationTimeoutSeconds(0.5),
+		prefab.WithContextTelemetryMode(options.ContextTelemetryModes.None))
```

Comment:
We spin up a number of clients in tests so I'm opting them out of telemetry where it isn't relevant